bundles/AdminBundle/translations/admin.en.yaml

```diff
@@ -882,9 +882,6 @@ batch_remove_selected: 'Batch remove from selected'
 batch_remove_from: 'Remove data from all'
 batch_remove_selected_from: 'Remove data from selected'
 unit_tooltip: 'Alternative units tooltip'
-toggle_image_features_visibility: 'Toggle Image Feature Visibility'
-detect_image_features: 'Detect Image Features'
-remove_image_features: 'Remove Image Features'
```

Comment:
This exists in all other translation files as well. We should probably remove it there too?

Reply:
Not necessary, they are auto-generated and will be cleared on the next import 😊